mx.py

```diff
@@ -2803,13 +2803,21 @@ def __eq__(self, other):
         else:
             return NotImplemented
 
+    def get_jdk_path(self, jdk, path):
+        # Exploded JDKs don't have a jre directory.
+        if exists(join(jdk.home, 'jre')):
```

Comment:
The issue here is, that mx.graal-core/suites.py links to the jre directory
```
    "JVMCI_API" : {
      "path" : "jre/lib/jvmci/jvmci-api.jar",
....
    },
```
in that case, following path is created: ``$JAVA_HOME/jre/jre/lib/jvmci/jvmci-api.jar``. We could do a bit of fuzzying like:

```
    def get_jdk_path(self, jdk, path):
        # Exploded JDKs don't have a jre directory.
        if exists(join(jdk.home, path)):
            return join(jdk.home, path)
        elif exists(join(jdk.home, 'jre')):
            return join(jdk.home, 'jre', path)
        else:
            return join(jdk.home, path)

```

this is not beautilful but in order to get this feature and have backwards compatibility we have to do it.
In the long run, we may remove the ``jre`` prefix from the suites.py dependencies.